src/pages/index.tsx

```diff
@@ -1,5 +1,38 @@
 import CampaignHero from "../components/hero/campaignHero";
+import {
+  SkipNavigation,
+  Template,
+  TemplateAboveHeader,
+  TemplateBreakout,
+  TemplateContent,
+  TemplateFooter,
+  TemplateHeader,
+} from "@nypl/design-system-react-components";
 
 export default function Home() {
-  return <CampaignHero />;
+  return (
+    <>
+      <SkipNavigation />
+      <Template>
+        <TemplateBreakout>
+          <TemplateAboveHeader>
+            <p> Notification banner </p>
+          </TemplateAboveHeader>
+          <TemplateHeader>
+            <p> Header </p>
+          </TemplateHeader>
+          <CampaignHero />
+        </TemplateBreakout>
+        <TemplateContent>
+          <p>First swim lane</p>
+          <p>Featured Content</p>
+          <p>Rest of swim lanes</p>
+          <p>Explore further links</p>
+        </TemplateContent>
+        <TemplateFooter>
```

Comment:
The NYPL Footer will go in a different location so this is okay to remove.

Reply:
Does this mean remove the template footer entirely?

Reply:
Yes, correct.